cogs/info.py

```diff
@@ -140,45 +143,70 @@ async def invite(self, ctx: commands.Context):
                        "<https://discord.com/oauth2/authorize?client_id=684395509045264429&permissions=378944&scope=bot>")
 
     @commands.command(description='Help Yuuto by giving us a suggestion or a bug report!', 
-                      help="This command will let you help Yuuto by giving it a suggestion or a bug report!", 
+                      enabled=suggest_enabled,
+                      help='This command will let you help Yuuto by giving it a suggestion or a bug report!', 
+                      usage='<message>',
                       aliases=['suggestion'])
     async def suggest(self, ctx: commands.Context, *, args: str = None):
-        suggestchannel: discord.TextChannel = await ctx.bot.fetch_channel(os.getenv('SUGGESTIONS_CHANNEL'))
+        try:
+            suggestchannel: discord.TextChannel = await ctx.bot.fetch_channel(705013080844926988)
+        except:
+            await ctx.send("An error occured: I can't find the suggestions channel")
+            print("Couldn't find suggestions channel!")
+            return
```

Comment:
This should probably disable the command, but I couldn't figure out a way to do it properly.

Reply:
I think that this is a good alternative